### ISSUE TITLE
Dedicated Task Queues

### ISSUE THREAD

**Issue by jokiefer** [mrmap-community/MrMap#36]
With an increasing number of apps using the background workers, we should think about creating dedicated queues for certain tasks. This should ensure that crucial tasks will not be blocked by time consuming lower priority tasks (e.g. metadata validation).

I suggest we should at least create dedicated queues for metadata validation and service registration, respectively.

What do you think @hollsandre ?
